packages/syft/src/syft/core/common/serde/deserialize.py

```diff
@@ -12,6 +12,10 @@
 from ....util import index_syft_by_module_name
 
 
+# WARNING: This code has more 🐉 Dragons than a game of D&D 🗡🧙🎲
```

Comment:
We need to handle this serde complexity jump as a tech debt story in the backlog.